### ISSUE TITLE
Reduce size of BalloonDescriber

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#392]
```
TODO: This file is massive. It should be substantially reduced in size.
```

It is currently 1907 lines. One way to break it up would be to have describers for different aspects of the balloon, like BalloonLocationDescriber and BalloonChargeDescriber. Some phrases require information about both, so BalloonDescriber could assemble these by using functions in each of BalloonLocationDescriber and BalloonChargeDescriber.

**Comment by jessegreenberg** [comment]
BalloonDescriberTests will also need to be changed.

**Comment by jessegreenberg** [comment]
BalloonDescriber has grown again after #416. Can we make it smaller?

**Comment by jessegreenberg** [comment]
Part of why it grew again is because we moved a lot of code out of BalloonNode and into BalloonDescriber, which is good.

**Comment by jessegreenberg** [comment]
I finalized this by improving on  the documentation in BalloonDescriber.js. I distributed logic about particular aspects of the balloon into BalloonChargeDescriber and BalloonLocationDescriber. But further abstraction doesn't feel helpful. Even though the file is still fairly large, I am going to accept that since it keeps all logic around balloon description in one place. Closing.
